rfcs/text/0001-rfc-process.md

```diff
@@ -15,6 +15,8 @@ As DCP continues to mature and scale, our community can more easily contribute a
 ## Author(s)
 *DCP PM*
 
+Arathi Mani + Dave Rogers (updated as of DD-MM-YYYY) 
```

Comment:
Please use the recommended format for authors from the template:

`[Name](mailto:username@example.com)`

Reply:
Done.